client/state/reader/streams/actions.js

```diff
@@ -105,9 +106,14 @@ export function fillGap( { streamKey, gap } ) {
 	} );
 }
 
-export function dismissPost( { streamKey, postId } ) {
+export const dismissPost = ( { streamKey, postKey } ) => {
+	wpcom
```

Comment:
could we do this with a direct http action instead? seems sad to pull in lib/wp just for this.  could be a follow up pr though.

Reply:
followup, don't want to do anything interesting in this quick fix

Reply:
I do love the idea though!